save-backend/src/test/kotlin/org/cqfn/save/backend/controller/ExecutionControllerTest.kt

```diff
@@ -153,13 +158,15 @@ class ExecutionControllerTest {
     @Test
     fun checkExecutionDto() {
         webClient.get()
-            .uri("/getDto?executionId=1")
+            .uri("/executionDto?executionId=1")
             .exchange()
             .expectStatus()
             .isOk
-            .expectBody<ExecutionDto>()
+            .expectBody<String>()
```

Comment:
You can expect body with specific class, it will use deserializer undre the hood